sections/main-product.liquid

```diff
@@ -1436,6 +1504,618 @@
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__31.label"
+            },
+            {
+              "value": "question_mark",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__32.label"
+            },
+            {
+              "value": "recycle",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__33.label"
+            },
+            {
+              "value": "return",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__34.label"
+            },
+            {
+              "value": "ruler",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__35.label"
+            },
+            {
+              "value": "serving_dish",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__36.label"
+            },
+            {
+              "value": "shirt",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__37.label"
+            },
+            {
+              "value": "shoe",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__38.label"
+            },
+            {
+              "value": "silhouette",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__39.label"
+            },
+            {
+              "value": "snowflake",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__40.label"
+            },
+            {
+              "value": "star",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__41.label"
+            },
+            {
+              "value": "stopwatch",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__42.label"
+            },
+            {
+              "value": "truck",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__43.label"
+            },
+            {
+              "value": "washing",
+              "label": "t:sections.main-product.blocks.collapsible_tab.settings.icon.options__44.label"
+            }
+          ],
+          "default": "heart",
+          "label": "t:sections.main-product.blocks.icon_with_text.settings.first_icon.label"
+        },
+        {
+          "type": "image_picker",
+          "id": "first_image",
+          "label": "t:sections.main-product.blocks.icon_with_text.settings.first_image.label"
+        },
+        {
+          "type": "text",
+          "id": "first_heading",
+          "default": "Heading",
+          "label": "t:sections.main-product.blocks.icon_with_text.settings.first_heading.label"
+        },
+        {
+          "type": "select",
+          "id": "second_icon",
```

Comment:
I'm not sure we've used words (first, second, third) instead of numbers for setting ids like this (correct me if you know of a case) Probably doesn't comes up a lot but I think we would usually call this `image_1`, `icon_1`, `heading_1`. Image banner is at least one place off the top of my head where I know this occurs (uses `image_1` and `image_2`) and I'd slightly prefer we keep consistent with that convention. I also just think the numbers are more concise and a little easier to parse when used in liquid conditionals and such.